analytics.js

```diff
@@ -919,6 +931,7 @@ class Analytics {
    */
   load(writeKey, serverUrl, options) {
     logger.debug("inside load ");
+    this.options = options;
```

Comment:
This can be slightly risky as the user can modify the options object after it is passed to this function. As we need it only for the consent manager, how about cloning it?

Reply:
Done